.ci/test_with_coverage.sh

```diff
@@ -1,23 +1,15 @@
 #!/usr/bin/env bash
 set -e
 
-dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-workdir=$dir/.cover
-profile="$workdir/cover.out"
+cover_dir="$(pwd)/.cover"
```

Comment:
nitpicking: cover_dir is now set based from where the script is executed and not the script location itself. If this is the desired behaviour, then, the $(pwd) is not needed.

Reply:
Yes you're right I'll fix that :slightly_smiling_face: